packages/libs/src/services/solana/transactionHandler.ts

```diff
@@ -280,23 +280,21 @@ export class TransactionHandler {
       })
     }
 
-    let txid
+    const txid = signatures?.[0]?.publicKey
```

Comment:
We should pull this from the transaction directly - not all callers are passing in the signature! Some of them rely on this method to sign for them